ivy/functional/frontends/jax/func_wrapper.py

```diff
@@ -60,16 +103,45 @@ def new_fn(*args, **kwargs):
 
 def outputs_to_frontend_arrays(fn: Callable) -> Callable:
     @functools.wraps(fn)
-    def new_fn(*args, **kwargs):
+    def new_fn(*args, order="K", **kwargs):
+        order = _set_order(args, order)
         # call unmodified function
         ret = fn(*args, **kwargs)
         # convert all arrays in the return to `jax_frontend.DeviceArray` instances
-        return _from_ivy_array_to_jax_frontend_array(
-            ret, nested=True, include_derived={tuple: True}
-        )
+        if order == "F":
+            return _from_ivy_array_to_jax_frontend_array_order_F(
+                ret, nested=True, include_derived={tuple: True}
+            )
+        else:
+            return _from_ivy_array_to_jax_frontend_array(
+                ret, nested=True, include_derived={tuple: True}
+            )
 
     return new_fn
 
 
 def to_ivy_arrays_and_back(fn: Callable) -> Callable:
     return outputs_to_frontend_arrays(inputs_to_ivy_arrays(fn))
+
+
+def outputs_to_frontend_arrays_with_order_manipulation(fn: Callable) -> Callable:
```

Comment:
this function `outputs_to_frontend_arrays_with_order_manipulation` seems identical to `outputs_to_frontend_arrays`, with the only difference being the addition of `order=order` when calling the original function `fn`. I would suggest we handle all of this inside the original `outputs_to_frontend_arrays`.

We can use the `inspect` module to see if `order` is a keyword argument of `fn`, and we can determine this inside the scope of `outputs_to_frontend_arrays ` but outside the scope of `new_fn`, which means this will only be called once when the function is first decorated. When will then have a boolean variable such as `contains_order` defined in the outer scope, which can be evaluated inside `new_fn`. Then we can simply do the following:

```
ret = fn(*args, order=order, **kwargs) if contains_order else fn(*args, **kwargs)
```

If `order` does not have any effect on the output (it is mathematically irrelevant for this function), then it's not a problem to pass it as input to `fn`, as it will just be ignored in that case anyway.

I might have missed something, but I think each of these `_with_order_manipulation` decorators can be removed, and we can instead generalise the original versions. Let me know if I missed something in my reasoning :)

Reply:
Fixed it!